src/main.ts

```diff
@@ -133,6 +133,12 @@ async function main() {
     const commits =
       eventObj.commits ||
       (await request(eventObj.pull_request._links.commits.href))
+    console.log('DEBUG: original commits', commits)
+    if (getBooleanInput('reverse')) commits.reverse()
```

Comment:
Are you sure this will be enough?
If I remember correctly, the problem was that the API was not returning more than 20 commits from that link: so if that's the case, this would reverse the 20 oldest commits, not return the 20 newest :/

I think the proper solution would switch to using the Octokit to make a proper API call to the REST GitHub API to get the actual list of commits. This way we should have **all** the commits, and there should be no need to reverse them.
Or, even better, we can process the diff for the whole commit range in `diff-search`, so that we're not checking each commit's diff, but only one.